src/redux/reducers/loader.ts

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState: { isLoading: boolean } = {
+  isLoading: false,
+};
+
+const loader = createSlice({
+  name: 'loader',
+  initialState,
+  reducers: {
+    show: (state): void => {
+      // eslint-disable-next-line no-param-reassign
+      state.isLoading = true;
+    },
+    hide: (state): void => {
+      // eslint-disable-next-line no-param-reassign
+      state.isLoading = false;
+    },
+  },
+});
+
+export default loader;
```

Comment:
On the outside we need only reducers, so you can export `loader.reducer`

Reply:
No, I'm using loader.reducer and loader.actions (in ReduxTestComponent component). The same way is in documentation.